scripts/patrol-local-integration-test-with-docker.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+## Pre-requisites
+# Install ngrok
+# Install patrol CLI
+# Open android emulator
+
+# Stoping previous environment if any
+killall ngrok || true
+cd backend-docker
+docker compose down || true
+cd ..
+
+# Forward traffic to tmail-backend
+ngrok http http://localhost:80 --log=stdout >/dev/null &
+until [[ $(curl localhost:4040/api/status | jq -r ".status") == "online" ]]; do
+    echo "Waiting for ngrok to connect..."
+    sleep 2
+done
+
+export BASIC_AUTH_URL=$(curl -s localhost:4040/api/tunnels | jq -r '.tunnels[0].public_url')
+
+cd backend-docker
+
+# Generate keys for tmail backend
+echo "Generating keys for tmail-backend..."
+openssl genpkey -algorithm rsa -pkeyopt rsa_keygen_bits:4096 -out jwt_privatekey
+openssl rsa -in jwt_privatekey -pubout -out jwt_publickey
+
+# Replace content of jmap.properties with url.prefix=$BASIC_AUTH_URL
+sed -i '' "s|url.prefix=.*|url.prefix=$BASIC_AUTH_URL|" jmap.properties
```

Comment:
Careful, `sed` on Linux and `sed` on macOS behave differently here.

For Linux, the syntax is:

```bash
sed -i "s|url.prefix=.*|url.prefix=$BASIC_AUTH_URL|" jmap.properties
```

While on macOS, the syntax is:

```bash
sed -i .bak "s|url.prefix=.*|url.prefix=$BASIC_AUTH_URL|" jmap.properties
```

See this answer on StackOverflow for more details: https://stackoverflow.com/a/4247319

Reply:
This script is only to run locally.